src/CTA.WebForms2Blazor/MigrationManager.cs

```diff
@@ -18,18 +18,20 @@ public class MigrationManager
         private WebFormsProjectAnalyzer _projectAnalayzer;
         private BlazorProjectBuilder _projectBuilder;
         private IEnumerable<FileInformation> _fileInformationCollection;
+        private FileInformationFactory fileFactory;
```

Comment:
If factory build methods are made static, we won't need to hold this variable here